lib/Cake/Test/Case/Utility/ValidationTest.php

```diff
@@ -1840,6 +1854,20 @@ public function testMinLength() {
 	}
 
 /**
+ * minLengthBytes method
+ *
+ * @return void
+ */
+	public function testMinLengthBytes() {
+		$this->assertFalse(Validation::minLengthBytes('ab', 3));
+		$this->assertFalse(Validation::minLengthBytes('ÆΔΩЖÇ', 11));
```

Comment:
no, mb_strlen() is utf8 character count. strlen() is 1 byte character count, that's why this would be working here.